tasks/package.js

```diff
@@ -61,29 +61,101 @@ module.exports = function(grunt) {
 
   grunt.registerTask('package', 'Package the operational codebase for deployment. Use package:compress to create an archive.', function() {
     grunt.loadNpmTasks('grunt-contrib-copy');
+    grunt.loadNpmTasks('grunt-shell');
 
     var config = grunt.config.get('config.packages');
-    var srcFiles = ['**', '!**/.gitkeep'].concat((config && config.srcFiles && config.srcFiles.length) ? config.srcFiles : '**');
+    var srcFiles = (config && config.srcFiles && config.srcFiles.length) ? config.srcFiles : [];
     var projFiles = (config && config.projFiles && config.projFiles.length) ? config.projFiles : [];
 
     // Look for a package target spec, build destination path.
     var packageName = grunt.option('name') || config.name || 'package';
-    var destPath = grunt.config.get('config.buildPaths.packages') + '/' + packageName;
+
+    // Determine if we are using rsync and vmPath and tarball for performance.
+    var useRsync = grunt.config('config.packages.rsync') || false;
+    // Determine if we are creating a tarball archive.
+    var archive = grunt.config('config.packages.archive') || useRsync;
+
+    // When using rsync, generate to a path that can be mounted in the VM.
+    var vmPath = grunt.config.get('config.packages.vmData') || 'build/vm_data';
```

Comment:
Per other thread, we're calling it tempDir. Just closing the loop here :) Certainly shorter than `intermediary`.